src/jsonapi-vuex.js

```diff
@@ -387,6 +391,27 @@ const jsonapiModule = (api, conf = {}) => {
 
 // Helper methods
 
+const cleanPatch = (patch, state) => {
+  if (!jvConfig.cleanPatch) {
+    return patch
+  }
+  const stateRecord = get(state, patch[jvtag]['id'])
+  if (!stateRecord) {
+    return patch
+  }
+  const clean = {}
```

Comment:
Could the construction of the patch (clean) be separated out into its own function? I can see situations where it would be useful to call it by hand (and regardless of whether jvConfig.cleanPatch is set).

Reply:
There are a number of these helper functions that are probably useful outside of the actions/mutations/getters... I'll see about refactorng this to allow it - then a future PR to export/make available these functions for general use.